src/core/config.py

```diff
@@ -1,15 +1,14 @@
 import os
-import google.generativeai as Client
 from github import Github
-from typing import Tuple
 
 class Config:
     GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
     GEMINI_API_KEY = os.environ.get('GEMINI_API_KEY')
     GEMINI_MODEL = os.environ.get('GEMINI_MODEL', 'gemini-1.5-flash-002')
+    OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+    OPENAI_MODEL = os.environ.get('OPENAI_MODEL', 'gpt-4o-mini')
     
     @classmethod
     def initialize_clients(cls):
```

Comment:
The pull request title and description lack sufficient detail.  Please provide a more descriptive title and description explaining the purpose of adding OpenAI model support and how it integrates with the `AIService` (assuming this is a class defined elsewhere).